src/v2i-hub/RSUHealthMonitorPlugin/manifest.json

```diff
@@ -19,7 +19,7 @@
         },
         {
             "key":"RSUConfigurationList",
-            "default":"{ \"RSUS\": [ { \"RSUIp\": \"192.168.XX.XX\", \"SNMPPort\": \"161\", \"AuthPassPhrase\": \"dummy\", \"User\": \"authOnlyUser\", \"RSUMIBVersion\": \"RSU4.1\" },{ \"RSUIp\": \"192.168.00.XX\", \"SNMPPort\": \"162\", \"AuthPassPhrase\": \"tester\", \"User\": \"authPrivUser\", \"RSUMIBVersion\": \"RSU4.1\" }] }",
+            "default":"{ \"RSUS\": [ { \"RSUIp\": \"192.168.XX.XX\", \"SecurityLevel\":\"authPriv\", \"SNMPPort\": \"161\", \"AuthPassPhrase\": \"dummy\", \"User\": \"authOnlyUser\", \"RSUMIBVersion\": \"RSU4.1\" ,\"RSUIdentifier\":\"NA\"},{ \"RSUIp\": \"192.168.00.XX\", \"SecurityLevel\":\"authPriv\", \"SNMPPort\": \"162\", \"AuthPassPhrase\": \"tester\", \"User\": \"authPrivUser\", \"RSUMIBVersion\": \"RSU4.1\" ,\"RSUIdentifier\":\"NA\"}] }",
```

Comment:
Is RSU Name or ID not the identifier? I think the RSU spec has snmp endpoints for both.

Reply:
RSUs specs does have RSU Name and ID. However, they usually have the same values across RSUs. We could use the RSUs specs, and that requires to update each individual RSUs setup either via SNMP request or some other ways. For the telematic tool purpose, I assume this is the least effort to meet the telematic requirement.